examples/resources/datadog_slo_correction/resource.tf

```diff
@@ -21,15 +21,15 @@ resource "datadog_slo_correction" "example_slo_correction" {
   description = "correction example"
   start       = 1735707000
   end         = 1735718600
-  slo_id      = "datadog_service_level_objective.example_slo.id"
+  slo_id      = datadog_service_level_objective.example_slo.id
   timezone    = "UTC"
 }
 resource "datadog_slo_correction" "example_slo_correction_with_recurrence" {
   category    = "Scheduled Maintenance"
   description = "correction example with recurrence"
   start       = 1735707000
-  rrule       = "FREQ=DAILY;INTERVAL=3;"
+  rrule       = "FREQ=DAILY;INTERVAL=3"
```

Comment:
same here, can we add count rule to the example? something like "FREQ=DAILY;INTERVAL=3;COUNT=3"